alpaca/data/live/websocket.py

```diff
@@ -2,22 +2,31 @@
 import logging
 import queue
 from collections import defaultdict
-from typing import Callable, Dict, Optional, Tuple, Union
+from typing import Callable, Dict, List, Optional, Tuple, Union
 
 import msgpack
 import websockets
 from pydantic import BaseModel
 
 from alpaca import __version__
 from alpaca.common.types import RawData
-from alpaca.data.models import Bar, Quote, Trade
+from alpaca.data.models import (
+    Bar,
+    Orderbook,
+    Quote,
+    Trade,
+    TradeCancel,
+    TradeCorrection,
+    TradingStatus,
+)
+from alpaca.data.models.news import News
```

Comment:
this News also can be a part of the import list above as we import news in models/__init__.py?